src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspace.SolutionCreator.cs

```diff
@@ -72,12 +72,6 @@ public async Task<Solution> CreateSolutionAsync(Checksum newSolutionChecksum)
                         Contract.ThrowIfFalse(solution.Id == newSolutionInfo.Id && solution.FilePath == newSolutionInfo.FilePath);
                     }
 
-                    if (oldSolutionChecksums.Options != newSolutionChecksums.Options)
-                    {
-                        var newOptions = await _assetProvider.GetAssetAsync<SerializableOptionSet>(newSolutionChecksums.Options, _cancellationToken).ConfigureAwait(false);
```

Comment:
do we even need SerializableOptionSet anymore?

Reply:
Yes, still need it. The beast has a lot of tentacles. Severing one at a time.